store/tikv/client.go

```diff
@@ -111,6 +113,28 @@ func (c *batchCommandsClient) failPendingRequests(err error) {
 	})
 }
 
+// recreate the gRPC stream.
+func (c *batchCommandsClient) recreateStreamLoop() {
+	for {
+		// try to re-create the streaming in the loop.
+		select {
+		case <-c.streamRecreateCh:
```

Comment:
If the recreate still fail, the code block here forever ?

Reply:
Yes, it will block until `batchSendLoop` wants to create the connection again.